src/main/java/edu/ucla/library/iiif/fester/CsvParser.java

```diff
@@ -304,6 +328,65 @@ private void extractPageMetadata(final String... aRow) throws CsvParsingExceptio
         return aRow;
     }
 
+    /**
+     * Checks for A/V metadata in rows that represent v2 canvases, and returns the row if none is found.
+     *
+     * @param aRow A row from the metadata CSV
+     * @param aPath A path to a CSV file
+     * @return The row
+     * @throws CsvParsingException If the row represents a v2 canvas and contains any A/V metadata
+     * @throws MimeTypeParseException
+     */
+    @SuppressWarnings("unchecked")
+    private String[] checkIiifPresApiCompatibility(final String[] aRow, final Path aPath, final String aIiifVersion)
```

Comment:
A very Java method name ;-)

I wonder whether `checkApiCompatibility` would have been enough, given the context of the application? No problem if the answer is "no" though.

Reply:
agreed, will change